main/src/ca/uwaterloo/flix/api/lsp/provider/FindReferencesProvider.scala

```diff
@@ -15,12 +15,14 @@
  */
 package ca.uwaterloo.flix.api.lsp.provider
 
-import ca.uwaterloo.flix.api.lsp.{Entity, Index, Location, Position}
+import ca.uwaterloo.flix.api.lsp.{Entity, Index, Location, LocationLink, Position}
```

Comment:
This is now a dead import :)

Reply:
Oh yeah, sorry for this, I cleaned up :)